brave/browser/password_manager/brave_password_manager_client.cc

```diff
@@ -228,23 +228,24 @@ bool BravePasswordManagerClient::PromptUserToSaveOrUpdatePassword(
   // "webby" URLs and do not prompt in case of "non-webby" URLS (e.g. file://).
   if (!CanShowBubbleOnURL(web_contents()->GetLastCommittedURL()))
     return false;
-  // FIXME(hferreiro)
-  // const autofill::PasswordForm *form = form_to_save->submitted_form();
-  // // Don't save password for confirmation page (ex. Trezor passphrase)
-  // if (IsPossibleConfirmPasswordForm(*form))
-  //   return false;
+  const autofill::PasswordForm* form =
+      static_cast<password_manager::PasswordFormManager*>(form_to_save.get())
```

Comment:
@darkdh An alternative here is to get `username_value` and `signon_real` from `form_to_save->GetPendingCredentials()`, but `IsPossibleConfirmPasswordForm()` still needs the `submitted_form()` because it checks its `layout` property. Any other ideas?

Reply:
use `GetPendingCredentials` for event emission and only static_cast for `IsPossibleConfirmPasswordForm`. This is probably the best way to deal with too simplified PasswordFormManagerForUI